plugins/woocommerce/src/Blocks/BlockTypes/Reviews/ProductReviewContent.php

```diff
@@ -0,0 +1,93 @@
+<?php declare( strict_types = 1 );
+namespace Automattic\WooCommerce\Blocks\BlockTypes\Reviews;
+
+use Automattic\WooCommerce\Blocks\BlockTypes\AbstractBlock;
+
+/**
+ * ProductReviewContent class.
+ */
+class ProductReviewContent extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'product-review-content';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 * @return string Rendered block content.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		if ( ! isset( $block->context['commentId'] ) ) {
+			return '';
+		}
+
+		$comment            = get_comment( $block->context['commentId'] );
+		$commenter          = wp_get_current_commenter();
+		$show_pending_links = isset( $commenter['comment_author'] ) && $commenter['comment_author'];
+		if ( empty( $comment ) ) {
+			return '';
+		}
+
+		$args         = array();
+		$comment_text = get_comment_text( $comment, $args );
+		if ( ! $comment_text ) {
+			return '';
+		}
+
+		/**
+		 * This filter is documented in wp-includes/comment-template.php
+		 *
+		 * @since 1.2.0
+		 */
+		$comment_text = apply_filters( 'comment_text', $comment_text, $comment, $args );
+
+		$moderation_note = '';
+		if ( '0' === $comment->comment_approved ) {
+			$commenter = wp_get_current_commenter();
```

Comment:
Isn't this variable defined above?

Reply:
Oh yeah good spot, I can't see why this was re-defined again. I've removed it in https://github.com/woocommerce/woocommerce/pull/57533/commits/2110255680f2912e577a5231b1f303fda530c2bb.